lib/main.js

```diff
@@ -54,22 +75,24 @@ module.exports = React.createClass({
   },
 
   render: function() {
+
```

Comment:
We should respect the formatting of the original library to prevent future merge conflicts